sky/templates/aws-ray.yml.j2

```diff
@@ -59,6 +59,18 @@ available_node_types:
           #   SpotOptions:
           #       MaxPrice: MAX_HOURLY_PRICE
       {% endif %}
+      UserData: |
+        #cloud-config
+        users:
+          - name: skypilot:ssh_user
+            shell: /bin/bash
+            sudo: ALL=(ALL) NOPASSWD:ALL
+            ssh-authorized-keys:
```

Comment:
https://cloudinit.readthedocs.io/en/latest/reference/examples.html#including-users-and-groups

Does `ssh-authorized-keys` work? The above link suggests `ssh_authorized_keys`.